pkg/AnewLibrary/Media/Atoms/DeviceUx.js

```diff
@@ -0,0 +1,132 @@
+export const atom = (log, resolve) => ({
```

Comment:
Extra credit for porting Media stuff!

Reply:
my pleasure. i'm going to do video next (i miss the video effects, they make me happy)))